demes/ms.py

```diff
@@ -989,8 +989,8 @@ def get_growth_rate(epoch):
             pulse = deme_or_pulse
             num_demes += 1
             new_deme_id = num_demes
-            e1 = Split(pulse.time, deme_id[pulse.dest], 1 - pulse.proportion)
-            e2 = Join(pulse.time, new_deme_id, deme_id[pulse.source])
+            e1 = Split(pulse.time, deme_id[pulse.dest], 1 - pulse.proportions[0])
+            e2 = Join(pulse.time, new_deme_id, deme_id[pulse.sources[0]])
```

Comment:
This looks like it will silently do the wrong thing when there are multiple pulse sources. I don't think it's too much work to support this, so feel free to do it if you're keen. Otherwise just raise a ValueError here if there are multiple sources, and we can open an issue to track it.

Reply:
Yes good point. I'll place a value error and open an issue for this. Will be easier to address in a smaller fix than here.